db/seeds.rb

```diff
@@ -63,11 +70,13 @@
   )
 end
 
-100.times do
-  idea = Idea.offset(rand(Idea.count)).first
-  user = User.offset(rand(User.count)).first
-  idea.comments.create!(
-    body: Faker::ChuckNorris.fact,
-    user: user
-  )
+puts 'Creating comment for each approved idea'
+Idea.find_each do |idea|
+  if idea.approved?
```

Comment:
It might be better to use factories and have one for `:idea, :with_comments` that takes a transient value for the number of comments, then call that a few times explicitly and remove the logic from the seeds.

Can leave this in for this PR but maybe worth doing another one for switching the seeds to use factories instead? :)

Reply:
Will leave for now and create a new PR for changing seeds to factories :)